src/lib/modules/client/terminal/ShortcutsHelp.svelte

```
@@ -1,114 +1,59 @@
 <script lang="ts">
+  import { KeyboardInput, Modal } from '@juspay/svelte-ui-components';
   import { getShortcutList } from './keyboard-shortcuts';
```

ESLint

[error] Missed spacing between "@juspay/svelte-ui-components" and "./keyboard-shortcuts".

(perfectionist/sort-imports)